configs/model.py

```diff
@@ -1,12 +1,13 @@
 DATASET_PATH = 'dataset'
 OUTPUT_PATH = 'dist'
+PICKLE_PATH = 'pickle'
```

Comment:
what artifacts does pickling make? could just write the weights to `dist/`?

Reply:
these bad boys, so basically preference on where to store them

![image](https://user-images.githubusercontent.com/33270038/203914335-1204821c-1da4-41d6-97c2-41f56912d555.png)